pkgs/development/tools/clickable/default.nix

```diff
@@ -1,43 +1,62 @@
-{ lib
-, fetchFromGitLab
-, buildPythonPackage
-, cookiecutter
-, requests
-, pyyaml
-, jsonschema
-, argcomplete
-, pytestCheckHook
-}:
+{ lib, fetchFromGitLab, buildPythonPackage, cookiecutter, requests, pyyaml
+, jsonschema, argcomplete, pytestCheckHook, watchdog, stdenv }:
 
 buildPythonPackage rec {
   pname = "clickable";
-  version = "7.4.0";
+  version = "7.11.0";
 
   src = fetchFromGitLab {
     owner = "clickable";
     repo = "clickable";
     rev = "v${version}";
-    sha256 = "sha256-QS7vi0gUQbqqRYkZwD2B+zkt6DQ6AamQO7sihD8qWS0=";
+    sha256 = "sha256-OVS+FK2ABoKbBFLDc3drcjeaa3yO9/8Ah8FzlN2fd8g=";
   };
 
-  propagatedBuildInputs = [
-    cookiecutter
-    requests
-    pyyaml
-    jsonschema
-    argcomplete
-  ];
+  propagatedBuildInputs =
+    [ cookiecutter requests pyyaml jsonschema argcomplete watchdog ];
```

Comment:
Please keep the original formatting, it looks nicer and introduces smaller diffs later. (Same above)